WooCommerce/WooCommerceTests/ViewRelated/Shipping Label/Create Shipping Label/ShippingLabelCustomPackageFormViewModelTests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+@testable import Storage
+
+class ShippingLabelCustomPackageFormViewModelTests: XCTestCase {
+
+    func test_properties_return_expected_values_for_empty_form() {
+        // Given
+        let shippingSettingsService = MockShippingSettingsService(dimensionUnit: "in", weightUnit: "oz")
+        ServiceLocator.setShippingSettingsService(shippingSettingsService)
+        let viewModel = ShippingLabelCustomPackageFormViewModel()
```

Comment:
Since we're already have `lengthUnit` and `weightUnit` in the intializer of `ShippingLabelCustomPackageFormViewModel` I think we can easily inject those without the need for a mock service?

Reply:
That's true. I was thinking that this way the test more closely mimics the expected behavior (getting those values from the service by default) but you're right that it isn't really needed here. Thanks!